msg/RadarTrack.msg

```diff
@@ -0,0 +1,23 @@
+# This message relates only to FMCW radar.  
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+# Object classifications (Additional vendor-specific classifications are permitted eg. 3 = Car)
+uint8 NO_CLASSIFICATION=0
+uint8 STATIC=1
+uint8 DYNAMIC=2
+
+
+uuid_msgs/UniqueID uuid                     # A unique ID of the object generated by the radar.
+
+                                            # Note: The z component of these fields is ignored for 2D tracking.
+geometry_msgs/Point position                # x, y, z coordinates of the centroid of the object being tracked.
+geometry_msgs/Vector3 velocity              # The velocity of the object in each spatial dimension.
+geometry_msgs/Vector3 acceleration          # The acceleration of the object in each spatial dimension.
+geometry_msgs/Vector3 size                  # The object size as represented by the radar sensor eg. length, width, height OR the diameter of an ellipsoid in the x, y, z, dimensions
+                                            # and is from the sensor frame's view.
+uint8 classification                        # An optional classification of the object (see above)
+float64[9] position_covariance              # Row-major about the x, y, z axes
```

Comment:
This is the current standard for covariance matrices [from REP 103](https://www.ros.org/reps/rep-0103.html) but it is very large. 

In the message API review it was deemed that a proposal to cut these down to lower resolution and upper triangular to cut the data size would be valuable: https://github.com/ros2/common_interfaces/issues/91 

Would people be interested in trying that out here. It's likely we can cut the covariance data size by two thirds just dropping to 32bit and upper triangular.

In this case it will save 192 of 288 bytes of covariance and will make this message overall about half the size per track. Since the current one has ~288 bytes of covariance and ~104 of position and uuid and one more for the classification. 

This would be a good amount of bandwidth savings.

Reply:
I'll yield to what you think is best on this front. The only criteria I think is important is consistency, so if other standard interfaces are going to move to this relatively soon, we can do that here too. If its not necessarily sure when that change would be made, I'd rather stick with tried and true.

Reply:
I have updated the covariance fields to float32's. I couldn't find any examples to copy about the wording to use for the upper triangle covariance matrix. Please review and suggest changes if required.